workers/unity/Packages/com.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -71,6 +75,29 @@ public static void Build()
             }
         }
 
+        public static void ValidateWorkerCanBuildForEnvironment(string workerType, BuildEnvironment targetEnvironment)
+        {
+            var spatialOSBuildConfiguration = SpatialOSBuildConfiguration.GetInstance();
+            var environmentConfig = spatialOSBuildConfiguration.GetEnvironmentConfigForWorker(workerType, targetEnvironment);
+            if (environmentConfig == null)
+            {
+                return;
+            }
+
+            var buildTargets = GetUnityBuildTargets(environmentConfig.BuildPlatforms);
+            var buildSupportResult = BuildSupportChecker.CheckBuildSupport(buildTargets);
+
+            if (buildSupportResult.CanBuild)
+            {
+                return;
+            }
+
+            var targetsString = string.Join(", ", buildSupportResult.TargetsWithoutBuildSupport);
+            throw new BuildFailedException(
```

Comment:
This is a straightforward check, why throw instead of return bool.

Reply:
Aye, simplified